nautobot/ipam/models.py

```diff
@@ -134,8 +141,10 @@ class VRF(PrimaryModel):
 
     clone_fields = [
         "tenant",
+        "enforce_unique",
         "description",
     ]
+    # TODO(jathan): Remove/revise this to account for Namespaces.
     natural_key_field_names = ["name", "rd"]  # default auto-key is just "rd", but it's nullable!
```

Comment:
Should be able to remove this actually, it'll default to the first `unique_together` (`["namespace", "name"]`) which should be fine.